Portfolio/index.html

```diff
@@ -1,16 +1,116 @@
+    <!-- Write your code below for PORTFOLIO and DO NOT TOUCH the code above-->
+    <!-- to import image from public folder, do it like this= <img src="../public/css.icon.png" alt="CSS image"> -->
+  </body>
+</html>
+<!DOCTYPE html>
+<html>
+    <head>
+        <title> <ul>PORTFOLIO</tittle>
+        <li>ABOUT</li>
+        <li>SKILL</li>
+        <li>PROJECTS</li>
+        <li>REVIEWS</li>
+        <br><br>
+        <body>
+        <main>  
+            <header>
+                <h2> AAYUSH SETIA</H2>
+                <img src="https://images.unsplash.com/photo-1506102383123-c8ef1e872756?q=80&w=2070&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D "width= "300px" alt=" this image not found ">
+                <h4> Frontend Developer</h4>
+            </header>
+                <h5>Get To Know Me more</h5>
+                <p>I m Skilled Front-end Developer with 4 years of experience in designing, developing and maintaining front-end web applications. Achieved measurable success in increasing page load speeds by 20% which lead to an increase in website conversion rates. Led the implementation of new front-end development system which resulted in a 15% reduction in development time and a 20% increase in website performance. Adept in collaborating with cross-functional teams and delivering projects on-time and on-budget</p>
+                <details> 
+                    <summary>information about me</summary>
+                    <p>Name = Aayush SETIA<br>
+                    experience= 2 years<br>
+                    Age=20</p>
+                </details>
+                <h2>SKILLES<H2/>
+                <i class="fa-brands fa-html5"></i>
+                <i class="fa-brands fa-css3-alt"></i>
+                <i class="fa-brands fa-js"></i>
+                <i class="fa-brands fa-react"></i>
+                <div class="container">
+                    <h2>connect with us </h2>
+                    <p> we would love to respond to your queries and help you</p>
+                    <div class="contact-box">
+                        <div class="contact-left">
+                            <h3> Send Your Request</h3>
+                            <form><div class="input-row">
+                                    <div class="input-group">
+                                        <label>Name</label>
+                                        <input type="text" placeholder="Aayush">
+                                        </div>
+                                        <div class="input-group">
+                                            <label>Phone number</label>
+                                            <input type="text" placeholder="+916280083340">
+                                            </div>
+                                            <div class="input-row">
+                                    <div class="input-group">
+                                        <label>email</label>

```

Comment:
The closing div tag for the main section should be moved to line 77 to properly close the section.